python/CMakeLists.txt

```diff
@@ -74,6 +74,9 @@ include(SetupCxxFlags)
 # Add common flags
 set(CMAKE_CXX_FLAGS "${CXX_COMMON_FLAGS} ${CMAKE_CXX_FLAGS}")
 
+# Enable perf and other tools to work properly
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer")
```

Comment:
This can affect performance, I'd rather have a target `ReleaseWithDebugInfo` that has `-fno-omit-frame-pointer -g`  set.